receiver/sshcheckreceiver/scraper.go

```diff
@@ -124,13 +125,15 @@ func (s *sshcheckScraper) scrape(ctx context.Context) (_ pmetric.Metrics, err er
 		}
 	}
 
-	return s.mb.Emit(), nil
+	s.rb.SetSSHEndpoint(s.Config.SSHClientSettings.Endpoint)
```

Comment:
Do you mind adding a test checking this gets indeed propagated?

Reply:
Done